.gitignore

```diff
@@ -17,7 +17,8 @@ b
 # Various DBs
 node-db*
 wallet-db/
-wallet-new-db/
+wallet-db-acid/
+wallet-db-sqlite
```

Comment:
This is the pattern I went for: instead of trying to come up with clever names for the new DB, considering now we have two of 'em, I am simply suffixing the name passed by the user (via the `--wallet-db-path`) with the proper DB. This ensure uniqueness with the old one, and give us a nice consistency.